### PR TITLE
LRQA-29748 Add SPA#SPANotificationSetTimeout poshi test

### PR DESCRIPTION
**Pull request by mateusralv**
Description:
SPA#SPANotificationSetTimeout

Original PR: 
https://github.com/liferay-frontend/liferay-portal/pull/3042

JIRA Ticket
https://issues.liferay.com/browse/LRQA-64578

### CODE REVIEW

**Comment by mateusralv** [comment]
ci:test:frontend:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/SPA.testcase [hunk 1]
```diff
@@ -35,4 +35,59 @@ definition {
 		AssertVisible.assertVisibleNoSPAWait(locator1 = "//html[contains(@class,'lfr-spa-loading')]");
 	}
 
+	@description = "LRQA-29748: Set the user notification timeout to 1 second"
+	@priority = 4
+	test SPANotificationSetTimeout {
+		property osgi.module.configurations = "userNotificationTimeout=1000";
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/SPA.testcase:41]: Do we need this property if we're already modifying it on the UI in System Settings?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/SPA.testcase [hunk 2]
```diff
@@ -35,4 +35,59 @@ definition {
...
+		}
+
+		task ("Then: notification timeout pop up immediately appears") {
+			AssertTextEquals(
+				locator1 = "ServerAdministrationScript#OUTPUT_FIELD",
+				value1 = "Timeout notification warning triggered");
+
+			WaitForElementNotPresent(
+				key_warningMessage = ${warningMessage},
+				locator1 = "Message#WARNING_SPECIFIC");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/SPA.testcase:89]: What is this assertion for?

**Review by john-co — APPROVED** [review]
Please see comments, besides that, looks good to me! :sunglasses:

**Comment by mateusralv** [comment]
ci:test:frontend:nocompile

**Comment by mateusralv** [comment]
ci:test:frontend:nocompile

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
The test passed in CI 🎉 

[LocalFile.SPA#SPANotificationSetTimeout](https://testray.liferay.com/home/-/testray/case_results/2228633907?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2228630806%26testrayRunId%3D2228632325%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**